src/color_tft.toit

```diff
@@ -116,6 +82,48 @@ COLOR_TFT_16_PIXEL_MODE_ ::= 5
 COLOR_TFT_18_PIXEL_MODE_ ::= 6
 COLOR_TFT_TRUNCATED_24_PIXEL_MODE_ ::= 7
 
+/**
+TrueColor Driver intended to be used with the Pixel-Display package
+  at https://pkg.toit.io/package/pixel_display&url=github.com%2Ftoitware%2Ftoit-pixel-display&index=latest
+See https://docs.toit.io/language/sdk/display
+
+ESP-WROVER_KIT v3 has the following pins:
+SDA:  23  (aka mosi)
+MISO: 25
+SCK:  19  (clock)
+CS:   22  (chip select)
+DC:   21  (command/data)
+TCS:   0  (touch screen chip select)
+RST:  18  (reset)
+BKLIT: 5  (backlight, active low)
+MADCTL: 0x40
+
+M5Stack:
+SDA:   23  (aka mosi)
+SCK:   18  (clock)
+CS:    14  (chip select)
+DC:    27  (command/data)
+RST    33  (reset)
+LIGHT: 32  (backlight)
+MADCTL: 0x00
+
+Lilygo T-Wristband (LSM9DS1 version):
+MOSI aka SDA: IO 19
```

Comment:
The 'aka' above is in parenthesis. Here you put it directly.

Here you use 'IO 19'. above you just have the numbers.

And for the lilygo below it's even "P" instead of "IO" or nothing.